{"name": "test_edit_flag", "status": "broken", "statusDetails": {"message": "AttributeError: 'YamlUtil' object has no attribute 'read_extract_yaml'", "trace": "self = <test_api.TestRequests object at 0x107643d90>\n\n    def test_edit_flag(self):\n        url = \"https://api.weixin.qq.com/cgi-bin/tags/update?access_token=\"\n>       value = YamlUtil().read_extract_yaml('access_token')\nE       AttributeError: 'YamlUtil' object has no attribute 'read_extract_yaml'\n\ntestcase/test_api.py:32: AttributeError"}, "start": 1645069570070, "stop": 1645069570070, "uuid": "b239a710-89ce-4b1e-ae37-e21461e9ef0a", "historyId": "451ab4afc2ba169bbb196d06ab1bb4c1", "testCaseId": "004f937a12f9de2824e2f61a6f17a04f", "fullName": "testcase.test_api.TestRequests#test_edit_flag", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestRequests"}, {"name": "host", "value": "YZQ.local"}, {"name": "thread", "value": "74376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_api"}]}